Dockerfile

```
@@ -25,6 +25,5 @@ FROM debian:buster-slim
 COPY --from=build-app /build/app/mmsd /app/
 
 WORKDIR /app
```

```suggestion
WORKDIR /app
RUN  chmod nobody.nobody /workdir
USER nobody:nogroup
```
Maybe this works? @shamlymajeed  can you try

---

RUN  chown nobody.nobody /workdir